components/MintPage/index.tsx

```diff
@@ -1,3 +1,3 @@
-import PausedMint from "./PausedMint"
+import MintV2Page from "./MintV2Page"
 
-export default PausedMint
```

Comment:
are we using `<PausedMint />` anymore?

can you delete the component if it's unused?

Reply:
removed this component 🚮